getting-started-with-recoil-in-react-1fca/notes/02_react-read-a-value-from-recoil-with-userecoilvalue.md.md

```diff
@@ -0,0 +1,11 @@
+<TimeStamp start="1:09" end="1:19">
+  
+  `useRecoilValue` is used to read state
```

Comment:
A link to the docs like you did in the previous lesson with useRecoilState would be good to have. Keeps things consistent and gives the user a way to get more info if they want.

Reply:
```suggestion
  `useRecoilValue` is used to read state. [recoiljs.org - useRecoilValue](https://recoiljs.org/docs/api-reference/core/useRecoilValue/)
```